<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>imageCanvasTool</class>
 <widget class="QWidget" name="imageCanvasTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>60</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">margin:0;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>206</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QToolButton" name="fullscreenBtn">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Fullscreen</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/insta360studio.qrc">
       <normaloff>:/player/fullscreen</normaloff>:/player/fullscreen</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="previewWindowBtn">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Preview</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/insta360studio.qrc">
       <normaloff>:/player/preview_windows_disabled</normaloff>:/player/preview_windows_disabled</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>36</width>
       <height>36</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="projectionModeBtn">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>View Projection</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/insta360studio.qrc">
       <normaloff>:/player/fisheye_mode_disabled</normaloff>:/player/fisheye_mode_disabled</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>36</width>
       <height>36</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="snapshotBtn">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Snapshot</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/insta360studio.qrc">
       <normaloff>:/player/snapshot_disabled</normaloff>:/player/snapshot_disabled</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>36</width>
       <height>36</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="deleteFileBtn">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Delete</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/insta360studio.qrc">
       <normaloff>:/player/delete_disabled</normaloff>:/player/delete_disabled</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>36</width>
       <height>36</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="MoreBtn">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>More Settings</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/insta360studio.qrc">
       <normaloff>:/player/setting_disabled</normaloff>:/player/setting_disabled</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>36</width>
       <height>36</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>206</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/insta360studio.qrc"/>
 </resources>
 <connections/>
</ui>
